core/vm/jump_table.go

```diff
@@ -1002,4 +1002,13 @@ func newFrontierInstructionSet() JumpTable {
 			maxStack:   maxStack(1, 0),
 		},
 	}
+
+	// Fill all unassigned slots with opUndefined.
+	for i, entry := range tbl {
+		if entry == nil {
+			tbl[i] = &operation{execute: opUndefined}
```

Comment:
Heh, a fun side-effect here. Since you didn't define `maxStack`, it's left at `0`, so this op will (mostly) never actually execute, it will always fail since the `maxStack` is reached (unless the stack is totally empty when this op is encountered). Setting maxStack to `maxStack(0,0)` would solve it

Reply:
Oh yes, I wanted this but did not realized I need to set non-zero maxStack (in my implementation I have "stack increase").